components/input-group/src/InputGroup.js

```diff
@@ -12,7 +13,17 @@ import { jsx } from '@westpac/core';
  * Input Group: Styled input fields with attached addons. Addons can be text ($, %, .00 etc) or form controls (buttons or select inputs).
  */
 export const InputGroup = ({ size, children, ...props }) => {
-	const childrenWithProps = Children.map(children, child => cloneElement(child, { size }));
+	const formContext = useContext(FormContext);
+
+	console.log('context', formContext);
+
+	const inputGroupSize = formContext.size || size;
```

Comment:
I think we should do this the other way around so the prop has precedence 

`const inputGroupSize = size || formContext.size || 'medium';

You will have to remove the default prop down below too